CurrencyExchangeServiceWCF/CurrencyExchangeServiceWCF/Service.svc.cs

```diff
@@ -85,50 +104,70 @@ private string GetActualConversionRate(string currency)
 
                         while (reader.MoveToNextAttribute())
                         {// Read the attributes.
-                            if (reader.Value.Equals(currency))
+                            if (reader.Value.Equals(Currency))
                             {
                                 reader.MoveToAttribute("rate");
                                 rate = reader.Value;
                             }
                         }
                         break;
-                    case XmlNodeType.Text: //Display the text in each element.
-                        break;
-                    case XmlNodeType.EndElement: //Display the end of the element.
-                        break;
-
+                         case XmlNodeType.Text: //Display the text in each element.
+                             break;
+                         case XmlNodeType.EndElement: //Display the end of the element.
+                             break;
+                             
                 }
             }
             rate = rate.Replace(".", ",");
             return rate;
         }
 
-        public List<string> GetCurrencyCodes()
+        public Response<List<string>> GetCurrencyCodes(GetCurrencyCodesRequest req)
         {
-            List<string> currencyList = new List<string>();
+            Response<List<string>> r = new Response<List<string>>();
```

Comment:
Please use the correct naming conventions

Reply:
fixed